.circleci/config.yml

```diff
@@ -25,14 +33,77 @@ jobs:
           apk update && apk upgrade
           apk add --no-cache git openssh make gcc g++ python
       - run:
-          name: Install dev dependencies
-          command: npm install
+          name: Install Dependencies
+          command: |
+            npm install
+            npm run lerna-reset
       - run:
-          name: Run linter
+          name: Run Linter
           command: npm run lint
       - run:
-          name: Bootstrap lerna
-          command: npm run lerna-reset
-      - run:
-          name: Run unit tests
+          name: Run Unit Tests
           command: npm run test
+
+  integration-test:
+    machine:
+      image: ubuntu-1604:201903-01 
+    steps:
+      - checkout
+      - run:
+          name: Setup Machine
```

Comment:
you probably don't need to install docker-compose

> The docker-compose utility is pre-installed in the CircleCI convenience images and machine executor


https://circleci.com/docs/2.0/docker-compose/

Reply:
nice! it does come pre-installed